### DISCUSSION TITLE
Welcome to valhalla-mobile Discussions!

### DISCUSSION THREAD

**Discussion by Archdoog** [Rallista/valhalla-mobile#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sourabh59-coder** [comment]
I have my own .pbf data that of mumbai(downloaded from [BBBikes](https://extract.bbbike.org/) ) tested by locally setting up the valhalla local server using docker desktop and it was returning the turn by turn route details with all the maneuvers, route and other details....i do have created the .tar file out of that .pbf data i want to know that how can i create the offline routing using that .tar or .pbf file of mine and run offline route generation in kotlin application....can you give me a walkthrough in short to follow to do this
